.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Dashboard team to review dashboard PRs
+/src/pybind/mgr/dashboard       @ceph/dashboard
```

Comment:
also `/qa/tasks/mgr/dashboard` ?

Reply:
You're right @sebastian-philipp. Thanks! BTW, you have already granted the approval "on behalf of @ceph/dashboard" (seems like the feature has magically started to work from the very PR) :astonished: